kibble/render/translations_api_renderer.go

```diff
@@ -33,6 +33,11 @@ func WriteLanguageFiles(site *models.Site, sourcePath string) error {
 		}
 
 		filename := fmt.Sprintf("%s.all.json", code)
+
+		if site.Translations[code] == nil {
```

Comment:
Better to check code exists on `site.Translations` to avoid panic.

```
data, ok := site.Translations[code]
if !ok {
	log.Errorf("Language '%s' contains no translations, check data exists.", code)
	// do we need to return an error here?
}

if data == nil {
	log.Errorf("Language '%s' contains no translations, check data exists.", code)
	// do we need to return an error here?
}
```

Reply:
@PeculiarGoat @stajs Updated for better logic, do we want to return errors here ? or are we wanting to have blank translations files, e.g en-au.all.json exist but empty ? cc @josejibin :)